### DISCUSSION TITLE
Source code for Question/set micro-service!

### DISCUSSION THREAD

**Discussion by Krishnaj20** [Sunbird-inQuiry/Community#5] [Documentation Issues]
Source code for Question/set micro-service is pointing to whole Knowledge platform!! Don't we have a separate building block called [Knowlg](https://knowlg.sunbird.org/). How's that separated out. It's confusing. I am hoping this will get fixed.

**Comment by Krishnaj20** [comment]
In AMJ quarter we will work on separating the Question/Question set service from Knowlg building block.

**Reply by pramodkvarma** [reply]
Ok, thanks @Krishnaj20 

@surendrasinghs until then DPGA submissions may not be acceptable with no clear installation guides, no clear separation on code, etc.
